index.html

```diff
@@ -1,10 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<meta charset="UTF-8">
-	<title>Document</title>
-<link rel="stylesheet" href="/assets/style.css" /></head>
-<body>
-	<h1>Hi</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="https://use.typekit.net/yri5dks.css" />
+    <title>Document</title>
+
+    
+  <link rel="stylesheet" href="/assets/style.css" /></head>
+  <body>
+    <div class="start-screen-block">
+      <header>
+        <ul>
+          <li><a href="#">HOME</a></li>
+          <li><a href="#">PORTFOLIO</a></li>
+          <li><a href="#">BLOG</a></li>
+          <li><a href="#">PAGES</a></li>
+          <li><a href="#">ABOUT US</a></li>
+          <li><a href="#">CONTACT</a></li>
+        </ul>
+      </header>
+      <div class="header-logo">
+        <img src="src/img/logo.png" style="width:50px;height:50px" />
+        <p>Expire</p>
+      </div>
+
+      <div class="header-slider">
+        <div class="arrows">
+          <img
+            src="src/img/left.png"
+            style="margin:120px -1px;position:absolute;"
+          />
+          <img
+            src="src/img/right.png"
+            style="margin-top:120px;position:absolute;right:1px;"
+          />
+        </div>
+        <p
+          style="font-family: raleway, sans-serif;
+			font-size:70px;
+		  font-weight: 700;
+		  
+		  font-style: normal;margin-top:-17px"
+        >
+          Expire
+        </p>
+        <img src="src/img/line.png" style="margin:-16px 95px;" />
+        <p style="font-size: 50px;width:910px">
+          Professionaly designed, carefully made for your enjoyement
+        </p>
+        <div class="header-buttons">
+          <button style="background:#D14B3A;border: 0px;">EXPLORE</button>
+          <button style="margin-left: 15px;">LEARN MORE</button>
+        </div>
+        <div class="slider-dots">
+          <img src="src/img/Dot_active.png" />
```

Comment:
It's better to make squares using css instead of images

Reply:
Catched! Done)